instrumentation/kafka-clients/src/main/java/brave/kafka/clients/KafkaTracing.java

```diff
@@ -72,6 +84,21 @@ public Builder remoteServiceName(String remoteServiceName) {
       return this;
     }
 
+    /**
+     * When no trace context or extra fields have been added to an incoming consumed message propagation,
```

Comment:
If an incoming message contains no trace context or extra fields, we create a new trace context. Normally, we add all the spans for a topic received in a batch into a single new trace. This flag can be used to instead force each message in the batch into its own trace.